Source/NSWindow.m

```diff
@@ -600,15 +600,17 @@ - (void) setTitle: (NSString*)aString
 {
   if (titleCell == nil)
     {
+      CGFloat fontSize;
+      
       titleCell = [[NSTextFieldCell alloc] initTextCell: aString];
       [titleCell setSelectable: NO];
       [titleCell setEditable: NO];
       [titleCell setBordered: NO];
       [titleCell setAlignment: NSCenterTextAlignment];
-      [titleCell setDrawsBackground: YES];
-      [titleCell setBackgroundColor: [NSColor blackColor]];
+      [titleCell setDrawsBackground: NO];
       [titleCell setTextColor: [NSColor whiteColor]];
-      [titleCell setFont: [NSFont systemFontOfSize: 8]];
+      fontSize = [NSFont systemFontSizeForControlSize: NSMiniControlSize];
```

Comment:
Isn't NSSmallControlSize more suited here?

Reply:
Maybe... I use the smallest type of font. I use NSSmallControlSize for labels like Workspace's FileViewer "10GB available on hard disk" label - it should be bigger and differ from the smalest one.